src/net.cpp

```diff
@@ -2869,8 +2869,8 @@ void CConnman::RecordBytesSent(uint64_t bytes)
         nMaxOutboundTotalBytesSentInCycle = 0;
     }
 
-    // TODO, exclude peers with download permission
-    nMaxOutboundTotalBytesSentInCycle += bytes;
+    if (increaseMaxOutbound)
+        nMaxOutboundTotalBytesSentInCycle += bytes;
```

Comment:
Per doc/developer-notes.md, brackets are generally required for conditionals over one line due to CVE-2014-1266, e.g. the Apple "goto fail" vulnerability, https://dwheeler.com/essays/apple-goto-fail
```diff
-    if (increaseMaxOutbound)
+    if (increaseMaxOutbound) {
         nMaxOutboundTotalBytesSentInCycle += bytes;
+    }
```